doc/mgr/orchestrator.rst

```diff
@@ -144,6 +144,11 @@ Example command::
 
      ceph orch device zap my_hostname /dev/sdx
 
+.. note::
+    If you have used "--all-available-devices" in OSDs creation, when you ``zap`` a device, immediately and automatically a new OSD will be created in the device.
+    To disable this behaviour, read information about the ``unmanaged`` parameter in :ref:`orchestrator-cli-create-osds`.
```

Comment:
s/ behaviour/behavior/

This is a bit of a nit-pick, but since Sage is American I proceed from the assumption that the documentation for Ceph is in American English. Since 1804, instances of "u" in words like "behavior" and "color" have been absent from American English.

Reply:
Thanks for the explanation! Learned! :-)